README.md

```diff
@@ -1,18 +1,26 @@
 # Building With Adobe DRM
 
+## Building the Application
+
 01. `git clone https://github.com/NYPL/Simplified-iOS.git` or `git clone git@github.com:NYPL-Simplified/Simplified-iOS.git`
 02. `cd Simplified-iOS`
 03. `git submodule update --init --recursive`
 04. Install [Cocoapods](https://cocoapods.org) if you haven't already.
 05. `pod install`
 06. Symlink the "DRM_Connector_Prerelease" directory to "adobe-rmsdk" within the "Simplified-iOS" directory. (You will need to have obtained the Adobe DRM Connector prerelease from Adobe.)
```

Comment:
Do we want to be helpful with a sample command?  "ln -s [your_path_here/]DRM_Connector_Prerelease adobe-rmsdk"